angularjs-example/tests/intern.js

```diff
@@ -1,60 +1,48 @@
-// Learn more about configuring this file at <https://github.com/theintern/intern/wiki/Configuring-Intern>.
-// These default settings work OK for most people. The options that *must* be changed below are the
-// packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
+// Learn more about configuring this file at <https://theintern.github.io/intern/#configuration>.
+// These default settings work OK for most people. The options that *must* be changed below are the packages, suites,
+// excludeInstrumentation, and (if you want functional tests) functionalSuites
```

Comment:
I guess we shouldn't mention benchmarkSuites there. It is a pretty new functional, I guess we should write a new example for benchmarks.